python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -107,7 +107,7 @@ def checkHostName():
         bash("chown cloud.cloud /var/run/cloudstack-management.pid")
         #distro like sl 6.1 needs this folder, or tomcat6 failed to start
```

Comment:
@remibergsma Could you point me to a couple lines of code where /var/log/cloudstack-management/ is used in  current master?